babel/messages/extract.py

```diff
@@ -38,7 +38,8 @@
     'dgettext': (2,),
     'dngettext': (2, 3),
     'N_': None,
-    'pgettext': ((1, 'c'), 2)
+    'pgettext': ((1, 'c'), 2),
+    'npgettext': ((1, 'c', 2, 3))
```

Comment:
I think the tuple should be `((1, 'c'), 2, 3)`?

I.e. that the first argument is `c`ontext and 2 and 3 are messages.